tests/unit/resources/filebeat.yml

```diff
@@ -0,0 +1,7 @@
+ # Wazuh - Filebeat configuration file
+ output.elasticsearch:
+ hosts: ["10.0.0.1:9200"]
+ protocol: https
+ username: admin
+ password: admin
+ 
```

Comment:
Is the extra white space at the beginning expected? I was confused when looking at the diff for a sec :smile:

Reply:
Wow! Me too haha. It seems there's a whitespace in line 7. Fixed